### ISSUE TITLE
RCT project output - Do we need Rule.evaluation_type

### ISSUE THREAD

**Issue by JasonGlazer** [open229/ruleset-model-description-schema#194]
During last schema WG call a question came up whether we need Rule.evaulation_type which points to an enumeration with values of FULL or APPLICABILITY. Need to come to decision.

**Comment by supriyagoel** [comment]
I don't have a strong opinion on this. A rule evaluated only for applicability will not result in pass/fail outcome, so the 'fully evaluated' rule doesn't provide additional value. 
However, when a rule is capable of being fully evaluated but results in an 'undetermined' outcome, it is a sign of missing data/inconsistencies. This also isn't any different from a  partially evaluated rule  resulting in undetermined outcome. 

tl;dr- i would be ok with removing this if the S-WG prefers it  that way

**Comment by mkarpman** [comment]
@supriyagoel , I think fully evaluated rule may produce "undetermined" outcome not only in case of missing/inconsistent data. For example, some fully evaluated rules have special conditions for vestibules, and since the RCT cannot always definitevly establish whether a particular space is a vestibule, it would produce undertermined outcome if a space may be a vestibule.

Aside from that, I think Rule.evaluation_type is helpful for identifying rules where the RCT exceeds the minimum requirements of 229 and can produce Pass/Fail outcomes for rules that are defined as "applicability only" by 229. We discussed at one of the committee meetings that it may be good to allow flaging such rules since the outcomes will not be tested (will not have associated TCDs). So if PASS or FAIL outcome is produced for a rule with rule.evaluate_type=APPLICABILITY, we can say that the outcome wasn't tested. Given that, I recommend keeping this field.

**Comment by JasonGlazer** [comment]
@weilixu what do you think?

**Comment by weilixu** [comment]
I would support to include the evaluation_type for the same reason Maria mentioned.

**Comment by JasonGlazer** [comment]
Sounds like we have reached a consensus. Rule.evaluation_type is already part of the RCT project output schema so no further action is needed.
